### ISSUE TITLE
[Textlink] Manage accessibility

### ISSUE THREAD

**Issue by robergro** [adevinta/spark-ios#772]
The textlink on UIKit and SwiftUI accessibility doesn't work. 
The component is not detected as a button by voice over and the label of the accessibility is not the good one
